src/lib.rs

```diff
@@ -74,7 +75,7 @@ impl Default for Config {
             trailing_args: vec![],
             host: None,
             target: None,
-            stderr_filters: vec![],
+            stderr_filters: vec![(Match::Exact(vec![b'\\']), b"/")],
```

Comment:
Hard-coding a default filter is kind of suspicious. Why is this needed?

Reply:
it's necessary, but not sufficient. This just gets rid of all the backslashes in stderr files without having to repeat that filter everywhere

Reply:
I'm a bit worried about having hard-coded default filters without an opt-out.

Reply:
we have an opt out, you can just overwrite the `stderr_filters` field, it's public.

Reply:
Oh I see. Sounds good then.

Reply:
But this should be documented somewhere.